WireMock/wwwroot/scripts/MatcherUI/HeaderMatcherUI.ts

```
@@ -37,12 +38,18 @@ export default class HeaderMatcherUI extends MatcherUI {
                 let matcherIndex = HeaderMatcherUI.getMatcherIndex(matcherRow);
                 let headerIndex = HeaderMatcherUI.getHeaderIndex(matcherRow);
                 rawMapContent.Request.Headers[headerIndex].Matchers.splice(matcherIndex, 1);
-                rawMap.textContent = JSON.stringify(rawMapContent, null, 1);
-                matcherRow.remove();
+                if (rawMapContent.Request.Headers[headerIndex].Matchers.length > 0) {
+                    rawMap.textContent = JSON.stringify(rawMapContent, null, 1);
+                    matcherRow.remove();
+                } else {
+                    this.showErrorModal("You cannot remove the last matcher - " +
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)